composer/loggers/mlflow_logger.py

```diff
@@ -171,16 +171,6 @@ def init(self, state: State, logger: Logger) -> None:
                     output_format='list',
                 )
 
-                # Check for the old tag (`composer_run_name`) For backwards compatibility in case a run using the old
```

Comment:
wait, we're not turning this off in this PR, right? @jerrychen109 that would just break resumption unless I'm misunderstanding something.

Reply:
I think we had talked about removing this shortly after we made the original change to rename the MLflow tag from `composer_run_name` to `run_name`. This was leftover from the original PR that we kept just to avoid breaking resumption for ongoing runs. There should be no runs started that are tagged with `composer_run_name` for the past 2 months, so unless there are 2-month-old runs that are ongoing and need to auto-resume with an updated composer version/commit, this should be safe? I'll leave it to your discretion though depending on how strictly we need to maintain backwards compatibility here, but I would be very very surprised if this breaks an actual user worklfow.

Reply:
ahh sorry. i misread what was being removed. thanks for clarifying, fine to delete